.eslintrc.js

```diff
@@ -1,26 +1,29 @@
 const path = require('path');
 
 module.exports = {
-  extends: ['airbnb', 'plugin:jest/recommended'],
+  extends: ['@titelmedia/eslint-config-es6', 'plugin:react/recommended', 'plugin:jest/recommended'],
   plugins: ['jest'],
   parser: 'babel-eslint',
   env: {
     browser: true,
+    node: true,
+    'jest/globals': true,
   },
   rules: {
-    'function-paren-newline': ['error', 'consistent'],
-    'import/no-extraneous-dependencies': ['error', { devDependencies: ['**/*.test.js*', 'config/**', 'tests/**'] }],
-    'import/extensions': false,
-    'import/no-extraneous-dependencies': 'off',
-    'jest/prefer-to-have-length': 'error',
-    'jest/prefer-to-be-null': 'error',
-    'jest/prefer-to-be-undefined': 'error',
-    'max-len': ['error', { code: 120, ignoreStrings: true, ignoreTemplateLiterals: true }],
-    'no-unused-expressions': ['error', { allowTernary: true }],    
-    'react/no-array-index-key': 'off',
-    'curly': ['error', 'all'],
+    'react/display-name': 'off', // TODO: fix once hs-lint supports react config
   },
+  overrides: [
+    {
+      files: ['**/*.test.js*', '**/*.spec.js*'],
```

Comment:
How come we're overriding the rules for test files?

Reply:
For tests we do not need to be that strict. For instance console.log and other stuff can be used there. In our production code users might see stuff. In general I agree, but this will be solved as soon as hs-lint introduces linting rules for react ;)